frameworks/cassandra/tests/test_overlay.py

```diff
@@ -3,6 +3,8 @@
 
 import shakedown
 
+# Do not use import *; it makes it harder to determine the origin of config
+# items
```

Comment:
:+1: Yes, this forever and everywhere. If we can't purge these mistakes easily, then marking them until they are fixed is a good compromise